tests/tests.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+
+from httplib import OK, NOT_FOUND
+import json
+import pkg_resources
+from nose.tools import assert_equal
+
+from openfisca_web_api.app import create_app
+
+TEST_COUNTRY_PACKAGE = 'openfisca_dummy_country'
+distribution = pkg_resources.get_distribution(TEST_COUNTRY_PACKAGE)
+subject = create_app(TEST_COUNTRY_PACKAGE).test_client()
+
+
+# /parameters
+
+parameters_response = subject.get('/parameters')
+
+
+def test_return_code():
+    assert_equal(parameters_response.status_code, OK)
+
+
+def test_package_name_header():
+    assert_equal(parameters_response.headers.get('Country-Package'), distribution.key)
+
+
+def test_package_version_header():
+    assert_equal(parameters_response.headers.get('Country-Package-Version'), distribution.version)
+
+
+def test_response_data():
+    parameters = json.loads(parameters_response.data)
+    assert_equal(
+        parameters[u'impot.taux'],
+        {u'description': u'taux d\'impôt sur les salaires'}
+        )
+
+
+def test_with_extra_slash():
+    response = subject.get('/parameters/')
+    assert_equal(response.status_code, NOT_FOUND)
```

Comment:
@fpagnoux [This stackoverflow answer](http://stackoverflow.com/a/33285603/3548266) explains you can set a global setting in flask: `app.url_map.strict_slashes = False` to accept trailing slashes.